backend/battling/forms.py

```diff
@@ -25,25 +24,26 @@ class Meta:
             "pokemon_3",
         ]
 
-    pokemon_1 = forms.ModelChoiceField(
+    pokemon_1 = forms.IntegerField(
         label="Pokemon 1",
-        queryset=Pokemon.objects.all(),
         required=True,
     )
-    pokemon_2 = forms.ModelChoiceField(
+    pokemon_2 = forms.IntegerField(
         label="Pokemon 2",
-        queryset=Pokemon.objects.all(),
         required=True,
     )
-    pokemon_3 = forms.ModelChoiceField(
+    pokemon_3 = forms.IntegerField(
         label="Pokemon 3",
-        queryset=Pokemon.objects.all(),
         required=True,
     )
 
     def clean(self):
         cleaned_data = super().clean()
 
+        cleaned_data["pokemon_1"] = save_pokemon(str(cleaned_data["pokemon_1"]))
```

Comment:
This name is also misleading. I think a better name would be `get_or_create_pokemon`